resources/language_server_node_eslint/launcher.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-exec ../nodejs/bin/node ./out/server.js --stdio
+exec ../nodejs/bin/node ./out/server.js --stdio  --max_old_space_size=70
```

Comment:
1. is it good enough or do you still call gc internally?
2. We should probably inform CDS to do the same...

Reply:
1. no. We need both.
2. You are correct. 
    Could you notify them?